src/builder/QuarkBuilder.sol

```diff
@@ -1019,18 +1164,32 @@ contract QuarkBuilder {
             }
             paymentTokenCost += nonBridgeAction.paymentMaxCost;
 
-            if (Strings.stringEqIgnoreCase(nonBridgeAction.actionType, Actions.ACTION_TYPE_TRANSFER)) {
-                Actions.TransferActionContext memory transferActionContext =
-                    abi.decode(nonBridgeAction.actionContext, (Actions.TransferActionContext));
-                if (Strings.stringEqIgnoreCase(transferActionContext.assetSymbol, paymentTokenSymbol)) {
-                    paymentTokenCost += transferActionContext.amount;
+            if (Strings.stringEqIgnoreCase(nonBridgeAction.actionType, Actions.ACTION_TYPE_BORROW)) {
+                continue;
```

Comment:
Why we don't have to worry about the cost for BORROW and WITHDRAW 🤔 ?

Reply:
because borrowing and/or withdrawing an asset increases your balance of that asset, whereas supplying that asset decreases your balance of that asset